hack/dockerfile/install/vndr.installer

```diff
@@ -1,8 +1,13 @@
 #!/bin/sh
 
-: "${VNDR_VERSION:=v0.1.2}"
+# FIXME(thaJeztah): temporarily using https://github.com/LK4D4/vndr/pull/95
+: "${VNDR_VERSION:=fix_vanity_url_handling}"
 
 install_vndr() {
+	set -e
 	echo "Install vndr version $VNDR_VERSION"
-	GOBIN="${PREFIX}" GO111MODULE=on go install "github.com/LK4D4/vndr@${VNDR_VERSION}"
+	git clone https://github.com/thaJeztah/vndr.git "$GOPATH/src/github.com/LK4D4/vndr"
+	cd "$GOPATH/src/github.com/LK4D4/vndr"
+	git checkout fix_vanity_url_handling
```

Comment:
Let's wait a short bit to see if @LK4D4 is around to get https://github.com/LK4D4/vndr/pull/95 merged (so that we don't have to use my fork)